src/tests/common.rs

```diff
@@ -74,7 +74,8 @@ mod inner {
     #[pymethods(crate = "pyo3")]
     impl UnraisableCapture {
         pub fn hook(&mut self, unraisable: &PyAny) {
-            let err = PyErr::from_value(unraisable.getattr("exc_value").unwrap());
+            let err =
+                PyErr::from_value_bound(&unraisable.getattr("exc_value").unwrap().as_borrowed());
```

Comment:
It might be nice to make `unraisable: &Bound<'_, PyAny>` here, to get a tidier end result immediately.